src/ResourceManager/Websites/Commands.Websites.Test/ScenarioTests/WebsitesTests.ps1

```diff
@@ -0,0 +1,72 @@
+﻿# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+<#
+.SYNOPSIS
+Tests creating a new website.
+#>
+function Test-CreatesNewSimpleWebsite
+{
+	# Setup
+	$rgname = "Default-Web-WestUS"
```

Comment:
You should not use static names like this - it does not scale and makes it impossible to point a test at a different environment.  Instead, use existing functions to get the locatiosn for your RP and prefer West US'